recipes/mp-units/2.0.0/conanfile.py

```diff
@@ -0,0 +1,158 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.59.0"
+
+
+class MPUnitsConan(ConanFile):
+    name = "mp-units"
+    homepage = "https://github.com/mpusz/mp-units"
+    description = "A Physical Quantities and Units library for C++"
+    topics = (
+        "units",
+        "dimensions",
+        "quantities",
+        "dimensional-analysis",
+        "physical-quantities",
+        "physical-units",
+        "system-of-units",
+        "system-of-quantities",
+        "isq",
+        "si",
+        "library",
+        "quantity-manipulation",
```

Comment:
```suggestion
        "quantity-manipulation",
        "header-only",
```

Reply:
It might be true for 2.0.0 but 2.1.0 will support C++ modules already which is hard to name "header-only".

Reply:
BTW, do you have any plans on how to handle and distribute C++ modules in Conan?

Reply:
Pinging @jcar87 which has some interesting stuff coming up on this front :)

Reply:
Hi @mpusz  - we are currently exploring how to best support modules and currently experimenting with a few libraries that have some support - if you have a preview of 2.1.0 with module support I can have a look and include that in our (internal, preliminary) testing.

First there's the question of what to package in the first place, then how it is consumed, and for the latter support is still needed in the wider tooling ecosystem - gcc only merged dependency scanning support last week (assuming this will make it to the gcc 14.1 release) and I expect CMake to have preliminary support for modules in imported targets in 3.28 - we'll try to very closely follow these developments so that for libraries that support modules, there are options  - even if they are experimental - so that tooling and library authors can make progress

Reply:
> if you have a preview of 2.1.0 with module support I can have a look and include that in our (internal, preliminary) testing.

I am about to merge the PR with modules (https://github.com/mpusz/mp-units/pull/350) in the next days. I will let you know when it is done,

Reply:
https://github.com/mpusz/mp-units/pull/350 now compiles!